### DISCUSSION TITLE
Import options

### DISCUSSION THREAD

**Discussion by Tarkusb** [keiwando/toonsquid#141] [Ideas]
Just to say, this app is wonderful, the work flow and stability are stellar. So far I’ve used it for two commissioned animations for websites. I refuse to buy into the adobe ecosystem and this is the first animation app on iPad that lets me achieve the results that I need. I think the progress this app is making is fantastic! 
The two things I would really like to see are SVG import and multiple clip import to library.
I made an animation recently using slices from affinity designer. So I’ve got a whole folder of 30 png’s that I wanted to use, had to import  them one by one, group import would be a real time saver.

**Comment by keiwando** [comment]
Thank you! I'm very happy to hear that :)
I am already planning on adding SVG import in the future and adding the ability to import multiple files at once is also on my list.
Would you want to be able to edit the SVGs using ToonSquid's vector tools or do you primarily just want to treat them as images? (keeping their vector nature and not getting rasterized, of course)

**Reply by kryptonianson** [reply]
If I may I would also like to add that both are very desirable. Being able to modify vector points would definitely be a major help to the animation process. Think about a bouncing ball, if it is SVG I can deform and animate it in-app.

My personal hope is that you add these features progressively so we can immediate value for new features as you implement. I would hope it would be something in this order:

- Multi-import
- SVG import
- SVG Editing
- SVG Save selected object back to library
- SVG Export
- SVG animating

I hope this helps. Thank you so much for an amazing product.

**Comment by Tarkusb** [comment]
Hi Keiwan,That’s great to hear you have these on the list!Hmm so I think both options could be useful, but the really great thing would be to edit vector points of imported svg’s using keyframes. I’ve been using the key framing a lot as I am intrinsically lazy 🙂. Are you thinking of adding some simple Boolean operations?, it would really save on layers, rather than grouping layers together. One other thing I would love to see is key framing of points in a vector brush line, great for things like waving grass etc.Just to say though, it’s great to able to do most of the animation in one app on iPad rather than the frankensteins monster approach I had to take before. And also…so stable and bug free.Thanks for getting back to me! CheersTarkusOn 6 Feb 2023, at 4:51 pm, Keiwan Donyagard ***@***.***> wrote:﻿
Thank you! I'm very happy to hear that :)
I am already planning on adding SVG import in the future and adding the ability to import multiple files at once is also on my list.
Would you want to be able to edit the SVGs using ToonSquid's vector tools or do you primarily just want to treat them as images (keeping their vector nature and not getting rasterized, of course)
